tests/js-units/data/bing_basemap-config.json

```diff
@@ -0,0 +1,206 @@
+            "title": "bing areals",
+            "abstract": "",
+            "link": "",
+            "minScale": 1,
+            "maxScale": 1000000000000,
+            "toggled": "False",
+            "popup": "False",
+            "popupSource": "auto",
+            "popupTemplate": "",
+            "popupMaxFeatures": 10,
+            "popupDisplayChildren": "False",
+            "popup_allow_download": true,
+            "legend_image_option": "hide_at_startup",
+            "groupAsLayer": "False",
+            "baseLayer": "False",
+            "displayInLegend": "True",
+            "singleTile": "True",
+            "imageFormat": "image\/png",
+            "cached": "False",
+            "clientCacheExpiration": 300,
+            "shortname": "Bing_Satellite",
+            "layerType": "raster",
+            "externalWmsToggle": "True",
+            "externalAccess": {
+                "crs": "EPSG:3857",
+                "format": "",
+                "type": "xyz",
+                "url": "https:\/\/ecn.t3.tiles.virtualearth.net\/tiles\/a{q}.jpeg?g=0&dir=dir_n'",
+                "zmax": "18",
+                "zmin": "1"
+            }
+        }
+    },
+    "timemanagerLayers": {},
+    "atlas": {
+        "layers": []
+    },
+    "tooltipLayers": {},
+    "layouts": {
+        "config": {
+            "default_popup_print": false
+        },
+        "list": []
+    },
+    "loginFilteredLayers": {},
+    "filter_by_polygon": {
+        "config": {
+            "polygon_layer_id": null,
+            "group_field": "",
+            "filter_by_user": false
+        },
+        "layers": []
+    },
+    "datavizLayers": {
+        "layers": [],
+        "dataviz": [],
+        "locale": "en_US"
+    },
+    "metadata": {
+        "qgis_desktop_version": 32815,
+        "lizmap_plugin_version_str": "4.1.8",
+        "lizmap_plugin_version": 40108,
+        "lizmap_web_client_target_version": 30800,
+        "lizmap_web_client_target_status": "Dev",
+        "instance_target_url": "http:\/\/localhost:8130\/",
+        "instance_target_repository": "intranet"
+    },
+    "options": {
+        "projection": {
+            "proj4": "+proj=merc +a=6378137 +b=6378137 +lat_ts=0 +lon_0=0 +x_0=0 +y_0=0 +k=1 +units=m +nadgrids=@null +wktext +no_defs",
+            "ref": "EPSG:3857"
+        },
+        "bbox": [
+            "398421.76870000001508743",
+            "5386390.87650000024586916",
+            "467349.17460000002756715",
+            "5427483.42289999965578318"
+        ],
+        "mapScales": [
+            1,
+            1000000000
+        ],
+        "minScale": 1,
+        "maxScale": 1000000000,
+        "use_native_zoom_levels": true,
+        "hide_numeric_scale_value": false,
+        "initialExtent": [
+            398421.7687,
+            5386390.8765,
+            467349.1746,
+            5427483.4229
+        ],
+        "bingKey": "AgBOKzHM1S17uF6szXY93vepXIx2Cl91mkrlP-yrsngz1jdREjrtucUduz56hDNT",
```

Comment:
is this key a dev key ? can it be exposed publicly  ?

Reply:
Hi @nworr,
I thought I could used it because it is already [exposed](https://github.com/3liz/lizmap-web-client/blob/2d836511de728cf24730a53ee98b0febbbd32ebb/tests/qgis-projects/tests/base_layers.qgs.cfg#L45) in `base_layers.qgs.cfg`. 

Thanks

Reply:
i didn't even notice this project use bing map :)